index.html

```diff
@@ -120,9 +120,18 @@ <h2 class="con-title">Contact me</h2>
         <p class="con-paragraph">If you have an application you are interested in developing, a feature that
           you
           need built or a project that needs coding. I’d love to help with it</p>
-        <form action="https://formspree.io/f/mnqlovzq" method="POST" class="con-form">
-          <input type="text" name="_name" placeholder="Name" maxlength="30" class="form-text" required>
-          <input type="email" name="_reply" placeholder="Please enter your Email" class="form-text" required>
+
+        <form novalidate id="form" action="https://formspree.io/f/mnqrnjol" method="POST" class="con-form">
+          <div class="input-control">
+            <input type="text" name="_name" placeholder="Name" maxlength="30" id="firstName" class="form-text" required>
+            <div class="error"></div>
+          </div>
+
+          <div class="input-control">
+            <input type="text" name="email" id="email" placeholder="Enter your Email" class="form-text">
+            <div id="error"></div>
```

Comment:
- [ ] Nice work you did on your validation, kindly check as a requirement the error message should show up close to the "submit" `button`.